templates/msgext/js/default/README.md

```diff
@@ -10,7 +10,7 @@ A Messaging Extension allows users to interact with your web service while compo
 
 ## Create an application
 
-- From Visual Studio Code, open command palette and select `Teams: Create New Project`. Or from the CLI, (after `npm install -g @microsoft/teamsfx-cli`) run command `teamsfx new`.
+- From Visual Studio Code, open command palette and select `Teams - Start a new project`. Or from the CLI, (after `npm install -g @microsoft/teamsfx-cli`) run command `teamsfx new`.
```

Comment:
Hi @IvanJobs, the latest command palette is "Teams: Create New Project", it should not be changed.